### ISSUE TITLE
Problem in sharing artwork

### ISSUE THREAD

**Issue by KawaiiZapic** [yellowbluesky/PixivforMuzei3#157]
Using "Share" button in Muzei does not pass a correct path to app. 
save to public storage is enabled.  
I can't catch any logs to give any information about this problem,I think this problem is caused by the removing of duplicate artwork in somewhere else and providing a not existed path to Muzei.

**Comment by yellowbluesky** [comment]
Thanks for bringing this to my attention. I'll have a look in the next few days.

**Comment by KawaiiZapic** [comment]
I spent some time to take a research on this feature, and I found that is not just a file path passing bug.  
I think there is a freaking mess in the code of storage.  
  
The "clean cache" button will not clean `/data/data/com.antony.muzei.pixiv/cache`,just clean `/sdcard/Android/data/com.antony.muzei.pixiv`,and after switch to next artwork,the old picture will still keep a copy in `/data/data/com.antony.muzei.pixiv/cache/muzei_com.antony.muzei.pixiv.provider`.
If enable save artwork to
